src/modules/graphics.c

```diff
@@ -128,6 +128,50 @@ CANVAS_ellipsefill(WrenVM* vm)
   ENGINE_ellipsefill(engine, x1, y1, x2, y2, c);
 }
 
+internal void
+CANVAS_triangle(WrenVM* vm)
+{
+  ASSERT_SLOT_TYPE(vm, 1, NUM, "x0");
+  ASSERT_SLOT_TYPE(vm, 2, NUM, "y0");
+  ASSERT_SLOT_TYPE(vm, 3, NUM, "x1");
+  ASSERT_SLOT_TYPE(vm, 4, NUM, "y1");
+  ASSERT_SLOT_TYPE(vm, 5, NUM, "x2");
+  ASSERT_SLOT_TYPE(vm, 6, NUM, "y2");
+  ASSERT_SLOT_TYPE(vm, 7, NUM, "color");
+  ENGINE* engine = (ENGINE*)wrenGetUserData(vm);
+  int64_t x0 = round(wrenGetSlotDouble(vm, 1));
+  int64_t y0 = round(wrenGetSlotDouble(vm, 2));
+  int64_t x1 = round(wrenGetSlotDouble(vm, 3));
+  int64_t y1 = round(wrenGetSlotDouble(vm, 4));
+  int64_t x2 = round(wrenGetSlotDouble(vm, 5));
+  int64_t y2 = round(wrenGetSlotDouble(vm, 6));
+
+  uint32_t c = round(wrenGetSlotDouble(vm, 7));
+  ENGINE_triangle(engine, x0, y0, x1, y1, x2, y2, c);
+}
+
+internal void
+CANVAS_trianglefill(WrenVM* vm)
+{
```

Comment:
Subtle formatting thing, the braces in this file are in the wrong place.

We keep them on the same line as the preceding syntax in the DOME project

Reply:
Fixed in a201791f71d355a08a97dc0183cb7359d044d7fa